app/service/app_svc.py

```diff
@@ -130,12 +133,22 @@ async def teardown(self, main_config_file='default'):
 
     async def register_contacts(self):
         contact_svc = self.get_service('contact_svc')
-        await contact_svc.register(Http(self.get_services()))
-        await contact_svc.register(Udp(self.get_services()))
-        await contact_svc.register(Tcp(self.get_services()))
-        await contact_svc.register(WebSocket(self.get_services()))
-        await contact_svc.register(Html(self.get_services()))
-        await contact_svc.register(Gist(self.get_services()))
+        for contact_file in glob.iglob('app/contacts/*.py'):
+            contact_module_name = contact_file.replace('/', '.').replace('\\', '.').replace('.py', '')
+            contact_module = import_module(contact_module_name)
+            classes_found = inspect.getmembers(contact_module, inspect.isclass)
+            for classname in range(0, len(classes_found)):
+                if not any(exclude in classes_found[classname][0] for exclude in ("BaseWorld", "Session", "Handle")):
```

Comment:
doing a little more in this block than you need. you can avoid looping through all the classes. instead, rename each class to "Contact" and that can be a requirement for contacts in the future.

Reply:
That makes life much easier! By changing the class names to "Contact" the register_contacts code more simple.
Changes are made in the commit below.